### DISCUSSION TITLE
(Detector Inference Speed) Frigate in Proxmox LXC with Codeproject-AI and Dual Coral TPU

### DISCUSSION THREAD

**Discussion by Bettman66** [blakeblackshear/frigate#12302] [Ask A Question] [answered]
### What is your question:

I'm using Codeproject.AI with Frigate 14 Beta3 in my Proxmox LXC environment for testing. I assigned the Dual Coral TPU to the Codeproject and have a detector inference speed of 14-17ms in Frigate. Since the network latency also has an effect, I think the value is correct, since I have about 6-7ms with the Coral in Frigate.
Can anyone confirm this?
![Screenshot 2024-07-05 064121](https://github.com/blakeblackshear/frigate/assets/15225986/a89c69bb-a70b-4797-b464-9de5fb39887f)

**Comment by NickM-27** [comment, marked as answer]
yes, makes sense
